controllers/fenceagentsremediation_controller.go

```diff
@@ -178,7 +177,7 @@ func (r *FenceAgentsRemediationReconciler) Reconcile(ctx context.Context, req ct
 			return emptyResult, fmt.Errorf("failed to remove finalizer from CR - %w", err)
 		}
 		r.Log.Info("Finalizer was removed", "CR Name", req.Name)
-		commonEvents.NormalEvent(r.Recorder, far, utils.EventReasonRemoveFinalizer, utils.EventMessageRemoveFinalizer)
+		commonEvents.NormalEvent(r.Recorder, far, utils.EventReasonRemoveRemediationCR, utils.EventMessageRemoveRemediationCR)
```

Comment:
Any particular reason for this change ? (EventReasonRemoveFinalizer -> EventReasonRemoveRemediationCR)
I think `EventReasonRemoveFinalizer` makes sense in this context.
Since FAR does not actually triggers the CR deletion I'd expect this sort of event to be triggered by NHC.

Also I can think of edge cases were the finalizer is removed without CR deletion being triggered.

Reply:
>as it could expect to see the AddFinalizer event as well

I agree, but I think there indeed should be an `AddFinalizer` event